src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ContainerDebugCommandTest.php

```diff
@@ -125,6 +126,19 @@ public function testTagsPartialSearch()
     public function testDescribeEnvVars()
     {
         putenv('REAL=value');
+
+        file_put_contents(__DIR__.'/app/.env', <<<EOF
+APP_EXAMPLE_ENV=example_value
+APP_EXAMPLE_ENV_TO_OVERRIDE=example_value
+EOF
+        );
+        file_put_contents(__DIR__.'/app/.env.local', <<<EOF
+APP_EXAMPLE_ENV_TO_OVERRIDE=example_overridden
+EOF
+        );
+
+        (new Dotenv())->bootEnv(__DIR__.'/app/.env');
```

Comment:
Instead of writing the `.env` files, you can simulate the behavior of `Dotenv` by setting the var `SYMFONY_DOTENV_VARS`.